### PR TITLE
CD pipeline improvements

### PR DESCRIPTION
**Pull request by KrzysztofLabus-RDX**

### CODE REVIEW

File: .github/workflows/releases.yml [hunk 1]
```diff
@@ -76,20 +82,27 @@ jobs:
...
 
       - name: Docker build and push Gateway API
         uses: docker/build-push-action@v2
         with:
           file: ./src/GatewayAPI/Dockerfile
-          TAGS: |
+          tags: |
             ${{ steps.setup_tags.outputs.gateway_api_tags }}
           push: true
+          platforms: linux/amd64,linux/arm64
```
Comment (diegomrsantos) [inline .github/workflows/releases.yml:107]: If I understand it correctly, this part is only about building the docker image. Was the application built with linux/arm64 support? I see only `--runtime linux-x64`.

Comment (diegomrsantos) [reply .github/workflows/releases.yml:107]: Is it necessary to build the application two times in the pipeline? If I understood it correctly it is being built during build and publish-docker?

Comment (KrzysztofLabus-RDX) [reply .github/workflows/releases.yml:107]: Should be enough according to the docs and to what I've tested locally using `docker buildx` - tool used under the hood by this workflow step.

When it comes to building: is it necessary? if we want to keep producing docker container and ZIP-archived release then short answer would be: yes, it is.

Comment (diegomrsantos) [reply .github/workflows/releases.yml:107]: I was wondering if the docker job could read the artifact created in the build job.

**Review by diegomrsantos — APPROVED** [review]

**Pull request merged**